woocommerce/woocommerce-ios
# [WOO POS] #13119 Update totals view design

Closes: #13119


## Description


Various updates according to p91TBi-btJ-p2

- Updated gradient colors (different gradient for payment successful state)
- Updated "new transaction" button
- Added corner radius to entire totals/checkout view
- Payments processing info views will be updated in separate PR(s)

## Testing information


- Enter POS
- Add few products
- Tap "Checkout"
- Check that the proper background gradient is show (purple)
- Check that the whole totals/checkout view has corner radius
- Check that the all amounts are calculated and properly shown
- Finish the payment
- Check that the "New transaction" button is show after the payment is successfully processed and that the background gradient is now green
- Tap on the "New transaction" button to start a new transaction

## Screenshots


![IMG_0296](https://github.com/woocommerce/woocommerce-ios/assets/6242034/978b39fa-53a5-451d-acea-489125b8924f)

![IMG_0297](https://github.com/woocommerce/woocommerce-ios/assets/6242034/d6e7e543-ef6f-4c22-a703-9a83d49310b4)


---
